lib/create-webpack-config.js

```diff
@@ -120,4 +106,23 @@ function createWebpackConfig(urc) {
   return urc.webpackConfigTransform(config);
 }
 
+function createBabelLoader(urc) {
+  const loader = {
+    loader: 'babel-loader',
+    options: {
+      presets: [require.resolve('../packages/babel-preset-mapbox')],
+      cacheDirectory: true,
+      babelrc: false,
+      compact: false
+    }
+  };
+  const exists = fse.pathExistsSync(path.join(urc.rootDirectory, '.babelrc'));
```

Comment:
`fs.existsSync` is a function. Do you think we can cut the `fs-extra` dependency?